### ISSUE TITLE
Piping command output into cm

### ISSUE THREAD

**Issue by adasium** [tsoding/cm#236]
Currently there is no support for piping into `cm` (e.g. `python main.py | cm`).
There are two reasons for it: one of them being the lack of possibility to repeat the command from inside of `cm`.
The other reason was a technical issue but I don't remember the details. Was it something tricky at the time so you decided to skip that for the time being or it's something difficult to work around and keep maintaining?

**Comment by rexim** [comment]
> The other reason was a technical issue but I don't remember the details.

Yeah, I think ncurses receives the input from `stdin`. And if you use `stdin` as the source of data ncurses can't work properly and you have to add hacks that read directly from `/dev/tty*` which is not crossplatform. I'll look into that again a bit later.

Thank you for filing this issue!
